examples/example_plugin/extended-viewer.js

```diff
@@ -0,0 +1,46 @@
+import $ from 'jquery'
+
+const highlightToggle = "red-highlight-toggle-on";
+const highlight = "example-containing-t-red-highlight";
+export class ExtendedViewer {
+    constructor(iv) {
+        this._iv = iv;
+    }
+
+    extendDisplayOptionsMenu = (menu) => {
+        menu.addCheckboxItem("Example Menu Item (Highlight Words containing the letter 'T' Red)",(checked) => {
```

Comment:
nit: splitting hairs, but this highlights all first level children elements of `body` whose `textContent` includes a "T". Meaning it highlights sections containing the letter 'T'